### ISSUE TITLE
Lets move phpPgAdmin forward and keep it well maintained!

### ISSUE THREAD

**Issue by flexjoly** [phpPgAdmin2/phpPgAdmin#1]
Hi,

I just created phpPgAdmin2 with the intention to add trusted contributors and maintainers, so we can keep this wonderful project alive.

PhpPgAdmin has no real alternatives. pgAdmin is totally different, and misses the features phpPgAdmin gives to work quickly with your Postgress Database.

Please respond here if you want to be a contributor and tell us why. Let us also vote a bit for we need trusted people to keep up the high standard.
I cannot be a real contributor myself. So when things workout great, I want to give the ownership to the ones who can maintain this project.

Creating a new organisation, with more members, makes it not-dependent on one person and gives us better possibilities to keep it alive and wellknown.

Greetz, flexJoly

**Comment by alexh-swdev** [comment]
Hi,

I don't want to slow down your effort (thanks for trying to keep this one maintained and up-to-date :) ) nor can I be of any help (lack of time) nor do I want to distract too much, but:

Why is PgAdmin4 no alternative? I was forced to switch recently due to missing phppgadmin maintenance. (thankfully, there are working docker images, so it was a very smooth transition) and did not yet miss any features. Some things like backups and restore are handled differently, though.
But I am also no "power user", just a software developer with basic db knowledge, no db admin.

**Comment by jaapmarcus** [comment]
We also have started to make a fork as our software depend on partially on phpgadmin in the past.

Feel free to pull any changes from:
https://github.com/hestiacp/phppgadmin

**Comment by N6REJ** [comment]
@jaapmarcus  I tried you 7.14.4 release and am gettrting the following both with php 8.1 & 8.2 
![image](https://user-images.githubusercontent.com/1850089/233238408-682fae2f-a176-4e75-8000-9ffe98b4aa3f.png)

**Comment by jaapmarcus** [comment]
We use our own config file:

https://github.com/hestiacp/hestiacp/blob/d4ae2dc1be57c39aaa5918bce904667851145e13/install/deb/pga/config.inc.php#L81-L88

Including this probably if enough

You maybe need to include the other options adding in the same commit

**Comment by jaapmarcus** [comment]
It is also available in:
https://github.com/hestiacp/phppgadmin/blob/ada77274a5042e2622c788ec0d3d94d33a5376b4/conf/config.inc.php-dist#L92-L99

**Comment by computersalat** [comment]
I really like OpenSource .. but I don't like to see several forks which confuse you which to follow.
Her is a Release mentioned:
* https://github.com/Bearsampp/module-phppgadmin/releases
wich does no exist here:
* https://github.com/phpPgAdmin2/phpPgAdmin/releases

can't we combine our efforts, please ?
See here: https://github.com/hestiacp/phppgadmin

**Comment by N6REJ** [comment]
> I really like OpenSource .. but I don't like to see several forks which confuse you which to follow. Her is a Release mentioned:
> 
> * https://github.com/Bearsampp/module-phppgadmin/releases
>   wich does no exist here:
> * https://github.com/phpPgAdmin2/phpPgAdmin/releases
> 
> can't we combine our efforts, please ? See here: https://github.com/hestiacp/phppgadmin

I agree completely.  So, I'm understganding your taking over phppgadmin responsibility?.

**Comment by computersalat** [comment]
I am a package maintainer and user. I would do so if the current maintainer agree.

**Comment by N6REJ** [comment]
> I am a package maintainer and user. I would do so if the current maintainer agree.

lets just come to some agreement over who's taking over so I know how to use & track.  We maintain a list/repo of sources we use so its kinda important to us.
https://github.com/Bearsampp/modules-untouched
